fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,25 @@
 */
 int fizzbuzz(int n)
 {
-
+int i;
+int counter = 0;
+  for( i = 1; i <= n; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
```

Comment:
Great job putting this conditional first! You can also check `if (i % 15 == 0)`.